src/ai/backend/manager/models/vfolder.py

```diff
@@ -843,8 +884,8 @@ async def load_slice(
         async with graph_ctx.db.begin_readonly() as conn:
             return [
                 obj
-                async for r in (await conn.stream(query))
-                if (obj := cls.from_row(graph_ctx, r)) is not None
+                for row in [r async for r in (await conn.stream(query))]
+                if (obj := cls.from_row(graph_ctx, row)) is not None
             ]
```

Comment:
These lines are not that readable. Can you flat out this block, avoiding inline list constructions so that everyone can easily read without any burden?